infrastructure/cdn-in-a-box/traffic_ops/to-access.sh

```diff
@@ -183,8 +182,7 @@ to-enroll() {
 	export MY_GATEWAY="$(route -n | grep $MY_NET_INTERFACE | grep -E '^0\.0\.0\.0' | tr -s ' ' | cut -d ' ' -f2)"
 	MY_NETMASK="$(ifconfig $MY_NET_INTERFACE | grep 'inet ' | tr -s ' ' | cut -d ' ' -f 5)"
 	export MY_NETMASK=${MY_NETMASK#"Mask:"}
-	MY_IP6_ADDRESS="$(ifconfig $MY_NET_INTERFACE | grep inet6 | grep -i global | sed 's/addr://' | awk '{ print $2 }')"
-	export MY_IP6_ADDRESS=${MY_IP6_ADDRESS%%/*}
```

Comment:
Why was this removed?

Reply:
Well it wasn't removed, exactly, it was just changed from defining the variable on one line and exporting it on the next.

I'm not sure exactly what the second line was supposed to do (@zrhoffman would know) but it was crashing the script so I moved the export up, removed that line and now it works.

Reply:
ok. Thanks for the explanation.